modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -53,6 +34,47 @@ spec:
 <1> Specifies the machine config pool label. 
 <2> Specifies the container runtime to deploy.
 
+* Optional: Create or edit an existing Security Context Constraint (SCC) to configure the `userNamespaceLevel` setting. This setting is optional, because the default value allows, but does not require, the use of user namespaces.
++
+.Example `SecurityContextConstraints` object definition
+[source,yaml]
+----
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: scc-admin
```

Comment:
maybe this should have a more informative name so it's clear which SCC we're choosing below? scc-userns or something?